test/jdk/java/lang/ProcessBuilder/Basic.java

```diff
@@ -2676,17 +2675,6 @@ static void THROWS(Class<? extends Throwable> k, Fun... fs) {
                 else unexpected(t);}}
 
     static boolean isLocked(BufferedInputStream bis) throws Exception {
```

Comment:
The original version in JDK 18 uses arbitrary Object and a configurable number of millis.  That said, this can stay as is as it's functionally the same as the original 18 test.